src/App.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <div id="app">
+    <Header 
+        v-bind:sort="sort"
+        v-bind:filter="filter"
+        v-bind:types="pokemonTypes"
+        v-bind:defense="defense"/>
```

Comment:
```suggestion
        />
```
Fixes Vue render error ("defense" not being passed down to Header component; no need for it)